float64 laser_range
float64 laser_intensity
float64 sonar_range
float64 deviation
bool is_water
